status/status.nim

```diff
@@ -30,7 +30,6 @@ type Status* = ref object
   settings*: SettingsModel
   mailservers*: MailserversModel
   tokens*: TokensModel
-  provider*: ProviderModel
```

Comment:
I know it is deprecated but you can also remove the file? if not, not big deal cause we are not using that in new architecture anyway

Reply:
It is still used in vendor/status-lib/status/updates.nim. I'll remove it once the ens content hash decode function is implemented! (i'm still missing that for the refactoring!)

Reply:
That is what I am working on!